pkg/cli/init.go

```diff
@@ -67,3 +70,40 @@ func InitConfig() {
 		logger.Infof("Using config file: %s", viper.ConfigFileUsed())
 	}
 }
+
+// SetupPrometheus sets up the prometheus server
+func SetupPrometheus(ctx context.Context, logger *zap.SugaredLogger, name string) error {
```

Comment:
nit: can grab the logger from the `ctx`, `logger := logging.FromContext(ctx)`

Reply:
It is being passed to the `func` from the server `logger := logging.FromContext(ctx)` https://github.com/naveensrinivasan/guac/blob/3e8614d14bb9585600e5ee1d6afc9f38ad6b7d57/cmd/guacgql/cmd/server.go#L70

Let me know